### DISCUSSION TITLE
Meeting Notes

### DISCUSSION THREAD

**Discussion by Jacinto-S** [B-Team-Organisation/Fantasy-Chess#6] [General]
Notes for our meetings.

## Template
```md
### Date: [Insert Date]
Attendees: [List of Attendees]

---

#### Agenda (Total Time: 1 hour):
- **00:00 - 00:10**: Agenda Item 1
- **00:10 - 00:20**: Agenda Item 2
- **00:20 - 00:40**: Agenda Item 3
- **00:40 - 01:00**: Agenda Item 4

---

#### Key Points Discussed:
- Key Point 1

---

#### Next Steps:
- Next Step


---

#### Action Items:
- [ ] Action Item

- [Team Member] Specific Action Item

```

**Comment by Jacinto-S** [comment]
### Date: 22.10.2024 8:15 - 13:15
Attendees: [List of Attendees]

Agenda:

- Bootstrapping
- First Prototyp

Key Points Discussed:

- [Plan](https://github.com/B-Team-Organisation/Fantasy-Chess/discussions/5#discussioncomment-11084240)
        Team Name: Das B-Team
        Game Idea: Turn Based Chess Spinoff
        Game Name: Fantasy Chess
        Requirements
        Architecture
        Scrum Roles
- [Prototyp](https://github.com/B-Team-Organisation/Fantasy-Chess/discussions/5#discussioncomment-11084240)
        Created carboard prototype
        

Next Steps:

- Intialize Github project with tech stack
- Set up project management

**Comment by Jacinto-S** [comment]
### Date: 27.10.2024 14:15 - 17:00
Attendees: Lukas Walker, Marc Matija, Jacinto Schwarzwälder

Agenda:

- Project setup and initial branch structure
- Framework integration

Key Points Discussed:

- GitHub Repository Structure:
        Repository initialized with main branch structure.
        Branches created for streamlined development flow: main, dev, and feature branches.
        Purpose of each branch:
            Main: Stable production-ready code.
            Dev: Ongoing development, to be merged into main upon completion.
            Feature: Isolated branches for new features, merged into dev after review.

- Framework Integration:
        LibGDX Framework: Basic project skeleton pushed to the repository, providing initial setup for game development.
        Spring Framework: Basic layout committed, establishing backend architecture and providing foundational components for future functionality.

Next Steps:

- Setup Project Management: Sprints, Milestones...
- Begin development on individual features in the feature branches.

**Comment by Jacinto-S** [comment]
### Date: 28.10.2024 15:00 - 17:00
Attendees: Lukas Walker, Marc Matija, Adnan Bennis, Dania Anwar, Jacinto Schwarzwälder

Agenda:

- Project management setup

Key Points Discussed:

- Configured Git project management
        General setup
        Added Boards, Kanban, Sprints
- Planned for MVP
        Created milestone
        Added issues
        Assigned story points
        Created [sequence diagram](https://github.com/B-Team-Organisation/Fantasy-Chess/discussions/5#discussioncomment-11084365)
- Discussed Techstack and Design
        Discussed results from previous meeting with the team
        First search for tileset (no fixed results)
        Some explorations of minor tech things (desuggering...)

Next Steps:

- Finalise planning, assign issues to members
- Onboarding of new member
- Begin development on individual features in the feature branches.

**Comment by casqan** [comment]
### Date: 29.10.2024
Attendees: Marc 

Agenda:

- Showing off progress to Igler (Checking off Checklist)
- Reviewing Planned Stories
- Working Environment
- Exam Registration
- Discussion Grading Criteria

Key Points Discussed:

- Key Points

Next Steps:

- Steps

Action Items:

- [x] [Team] Milestone in Sprint View zeigen
- [x] [Team] Grobplanung: Stunden ausrechnen, konkretisieren
- [x] [Team] Projektbeschriebung vervollständigen
- [ ] [Team] FMC Überarbeiten: Deskriptiver analog Sequenzdiagram, Spielfeld auch in DB schmeißen
- [x] [Team] Sequenzdiagramm: Schleife einfügen, etwas überarbeiten
- [x] [Mit Igler] Product Backlog ausführlicher durchgehen
- [x] [Team] Kursanmeldung nicht vergessen!

**Reply by Jacinto-S** [reply]
20h / Woche

**Reply by Jacinto-S** [reply]
# Zur Doku
- Installation
- Betrieb
- Weiterentwicklung
Für Abschlussvortrag:
- Scrum: Je Sprint -> Sprint Review, Sprint Retrospective
- Projektverlauf, was ist gut und schlecht gelaufen...

**Comment by Jacinto-S** [comment]
### Date: [04.11.2024]
Attendees: Marc, Jacinto, Lukas, Adnan, Albano, Dania

Agenda:

- FMC Diagram, Product Backlog für Igler
- Current Progress on Project
- Plan for centralized Documentation
- Moodboard and Graphics
- Clockify Review

Key Points Discussed:

- Update Backlog
- Testing coverage, to not test logic too overkill
- Guidelines
- Tileset and Graphics

Next Steps:

- Steps

Action Items:

- [x] Check Clockify
- [x] Stories for Character & Board Graphics
- [x] Update guidelines for squash commit to feature a Link to Issue
- [x] Check Testing coverage in Pull Request
- [x] Figma Main Menu Concept
- [ ] Redo FMC Diagram
- [x] Overhaul and refine issues

**Comment by Jacinto-S** [comment]
### Date: 11.11.2024
Attendees: Entire Team

---

#### Agenda (1-hour Total):
- **00:00 - 00:10**: Team Updates
- **00:10 - 00:20**: Review Unchecked Action Items
- **00:20 - 00:40**: Check Current State of Game (MVP Progress)
- **00:40 - 01:00**: **Overhaul Issues** (Very High Priority)
- **01:00 - finish**: Discuss ideas etc.

---

#### Key Points Discussed:
- Whoever is late without announcing it before must bring a pack of cookies next time.
- Update from every team member
- Quick review of unchecked action items
- Issue Overhaul

---

#### Next Steps:
- Sprint Planing

---

#### Action Items:
- [ ] FMC Diagram

**Comment by casqan** [comment]
### Date: 12.11
Attendees: Marc, Igler

---

#### Key Points Discussed:
- Issues
- Current Progress

---

#### Next Steps:
- Prepare Presentation for other groups

---

#### Action Items:
- [x] Get Project Running on Igler's PC

**Comment by adbennis** [comment]
### Date: 18.11
Attendees: Marc, Jacinto, Lukas, Adnan, Albano, Dania

---

#### Agenda (Total Time: 45 minutes):
- **00:00 - 00:25**: Update the issues for clarification
- **00:25 - 00:45**: Discuss if issues are clear for everyone

---

#### Key Points Discussed:
- explain what everyone understand by his issues
- discuss ideas

---

#### Next Steps:

- [x] See how to get the project running on Iglers PC

**Comment by adbennis** [comment]
### Date: 25.11
Attendees: Marc, Jacinto, Lukas, Adnan, Albano, Dania

---

#### Key Points Discussed:
- Discuss how to implement Turn Logic
- clear uncertainties about some issues

---

#### Next Steps:
- Sprint Planing 
- Character conception

**Comment by Jacinto-S** [comment]
### Date: [02.12.2024]
Attendees: [Lukas, Jacinto, Albano, Marc]

---

#### Agenda (Total Time: 1 hour):
- **00:00 - 00:10**: Agenda Item 1
- **00:10 - 00:20**: Agenda Item 2
- **00:20 - 00:40**: Agenda Item 3
- **00:40 - 01:00**: Agenda Item 4

---

#### Key Points Discussed:
- Status Update from everyone
- Marc: Networking, Main Menu...
- Jacinto: Character Representation, Excel Sheet for Issue Time eval, Turn Logic is blocking and is in progress
- Lukas: Character Representation, Command Preview done, Refactoring, next: Game Initialization & characters,...
- Lukas & Marc will Design Characters this week
- Albano: Turn Logic, Visualisation context, game summary, made 2 characters
---

#### Next Steps:
- After MVP: Documentation, CI/CD, Refactoring

---

#### Action Items:
- [ ] Refactoring / Documentation Sprint after MVP

- [Team Member] Specific Action Item

**Comment by Jacinto-S** [comment]
### Date: [09.12.2024]
Attendees: [Lukas, Marc, Dania, Albano, Jacinto, Adnan]

---

#### Agenda (Total Time: 1 hour):
- Update
- Planning

---

#### Key Points Discussed:
- Updates from each team member (characters missing..., Game Initialization waiting for command management UI, which is WIP and characters are mocked will need from server, some lobby stuff missing but wip, backlog bug with client not disconnecting added, turn outcome visualisation)
- Albano takes over the issue "Damage Indicator & Health Visualisation"
- Next Sprint Planning after tuesday presentation
- Discuss if we should increase the pixel size of the Map
- temporary game summary screen for the MVP 
- mirroring characters to make them recognizable on the board 

---

#### Next Steps:
- Sprint Planning
- Update character templates


---

#### Action Items:
- [ ] Finish 5 Characters
- [ ] Game summary screen

**Comment by adbennis** [comment]
### Date: 14.12.2024
Attendees: Jacinto, Lukas, Marc, Adnan

---

#### Key Points Discussed:
- Get an overview of the project
- Sprint planning
- Everyone can select the issue they want

---

#### Next Steps:
- Finish the MVP


---

#### Action Items:
- [ ] Finish MVP till the end of the Next week

**Comment by adbennis** [comment]
### Date: 16.12.2024
Attendees: Marc, Lukas, Albano, Adnan

---


#### Key Points Discussed:
- Status Update from everyone
- Design descussions
- Discuss new issues 
---

#### Next Steps:
- finish the MVP

**Comment by adbennis** [comment]
### Date: 23.12.2024
Attendees: Lukas, Dania, Albano, Jacinto, Adnan

---

#### Key Points Discussed:
- Discuss new plan over the holidays
- Assign new issues 
- Clarification of single issues 
- Plan the presentation for the game day in the next sprint 

---

#### Next Steps:
- Finish all the remaining issues in the issueboard till the 30.12.24
- Sprint planning next week at 2pm


---

#### Action Items:
- [ ] Finish all the remaining issues in the issueboard till the 30.12.24
- [ ] Have a playable game for the Game day (08.01.25)
- [ ] Video and Poster for the Game day

**Comment by Jacinto-S** [comment]
### Date: 30.12.2024
Attendees: Lukas, Marc, Albano, Jacinto

---

#### Key Points Discussed:
- Sprint will continue as is 

---

#### Next Steps:
- Continue & finish Sprint

**Comment by adbennis** [comment]
### Date: 06.01.2025
Attendees: Marc, Albano, Jacinto, Lukas, Dania, Adnan

---

#### Key Points Discussed:
- finish the video and poster by 08.01
- assign remaining issues 

---

#### Next Steps:
- Playtest the game
- Final touches to finish the MVP 
- 


---

#### Action Items:
- [ ] Playtesting the game

**Comment by adbennis** [comment]
### Date: 13.01.2025
Attendees: Adnan, Marc, Lukas, Dania, Jacinto, Albano

---

#### Key Points Discussed:
- Discuss our presentation on Game day
- Which Server are we using for the Game day

---

#### Next Steps:
- finish all issues till wednesday

---

#### Action Items:
- [ ] Game day on Wednesday

**Comment by casqan** [comment]
### Date: 20.01.2025
Attendees: Jacinto, Albano, Marc

---

#### Key Points Discussed:
- Feature Stop
- Only finish issues that are in progress, do not start new features
- look over documentation and document what is missing.

---

#### Next Steps:
- Overhaul Documentaton
- fix bugs
- finish in progress issues

**Comment by adbennis** [comment]
### Date: 27.01.2025
Attendees: Marc, Adnan, Lukas, Dania, Jacinto

---

#### Key Points Discussed:
- wont start working on new issues 
- only bugfixes
- discuss how we handle the presentation
- everyone documents the things they have done

---

#### Next Steps:
- start documentation
- prepare presentation


---

#### Action Items:
- [ ] practice presentation on 02.02 online and 04.02 in person
- [ ]
